### PR TITLE
Feature 109 complex video elements

### LINKED ISSUE #109: Complex video elements

### PR DESCRIPTION
**Pull request by villejuutila**
Resolves #109

### CODE REVIEW

File: src/components/layout/v2/grouped-inputs-with-lock.tsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ const GroupedInputsWithLock = ({ type, onChange, styles }: Props) => {
               value={(parseInt(styles[name] || styles[type]) || "0").toString()}
               name={name}
               inputProps={{
-                pattern: "[0-9]"
+                pattern: "-[0-9]"
```
Comment (anttileppa) [inline src/components/layout/v2/grouped-inputs-with-lock.tsx:82]: This pattern matches only texts where first character is - and next one is number, for example -2 but it does not match for -22 or 22 or 2.

You are using parseInt in the value, so it anyway enforces the value to be int, so I would just remove whole pattern

File: src/components/layout/v2/video-component-properties.tsx [hunk 2]
```diff
@@ -76,62 +89,23 @@ const VideoComponentProperties = ({
...
-        break;
-      case "autoplay":
-        element.autoplay = checked;
-    }
-
-    updateComponent({
-      ...component,
-      element: element
-    });
+  const getVideoElement = () => {
```
Comment (anttileppa) [inline src/components/layout/v2/video-component-properties.tsx:92]: Add docs

**Review by anttileppa — APPROVED** [review]

File: src/components/layout/v2/grouped-inputs-with-lock.tsx (hunk 1 above)
Comment (villejuutila) [reply src/components/layout/v2/grouped-inputs-with-lock.tsx:82]: True. Resolved.

File: src/components/layout/v2/video-component-properties.tsx (hunk 2 above)
Comment (villejuutila) [reply src/components/layout/v2/video-component-properties.tsx:92]: Resolved

**Pull request merged**